frontend/components/site/siteBuildLogs.js

```diff
@@ -7,20 +7,18 @@ class SiteBuildLogs extends React.Component {
   }
 
   buildLogs() {
-    if (!this.props.buildLogs) {
+    if (this.props.buildLogs.isLoading || !this.props.buildLogs.data) {
       return [];
+    } else {
+      return this.props.buildLogs.data
     }
-
-    return this.props.buildLogs.filter(log => {
-      return log.build.id === parseInt(this.props.params.buildId);
-    }).sort((a, b) => {
-      return new Date(a.createdAt) - new Date(b.createdAt);
-    });
   }
 
   render() {
     if (this.buildLogs().length > 0) {
       return this.renderBuildLogsTable()
+    } else if (this.props.buildLogs.isLoading) {
+      return this.renderLoadingState()
     } else {
       return this.renderEmptyState()
```

Comment:
I think it's a little interesting we did
```
if (this.props.publishedFiles.isLoading) {
       return this.renderLoadingState()
} else if (!files.length) {
      return this.renderEmptyState()
} else {
      return this.renderPublishedFilesTable(files)
}
```
for site published files and broke down the three-part condition differently here. I'd pick whichever one makes sense and go with it.